install/windows/viewer.spec

```diff
@@ -58,6 +60,10 @@ def get_dataio_imports():
     return imports
 
 
+def get_wx_imports():
+    return collect_submodules('wx.lib.pubsub.core.kwargs') + collect_submodules('wx.lib.pubsub.core.arg1') + ["wx.lib.pubsub.core.publisherbase"] + ["wx.lib.pubsub.core.listenerbase"]
```

Comment:
Please make it more readable by setting it on multiple lines and directly as `["wx.lib.pubsub.core.publisherbase", "wx.lib.pubsub.core.listenerbase"]`.

Also, mention in a comment that the special hooks-wx.lib.pubsub doesn't do its job properly.